Docs/Configuration.tex

```diff
@@ -2765,126 +2762,129 @@ \section{Misc}\label{misc}
+algorithm.
+
+The display order as follows:
 
 \begin{itemize}
 \tightlist
-\item Alternate options follow corresponding primary options, i.e. Apple recovery will be following the
+\item Alternate options follow corresponding primary options. That is, Apple recovery options will follow the
   relevant macOS option whenever possible.
-\item Options will be listed in file system handle firmware order to maintain an established order across
-  the reboots regardless of the chosen operating system for loading.
+\item Options will be listed file system handle firmware order to maintain an established order across
+  reboots regardless of the operating system chosen for loading.
 \item Custom entries, tools, and system entries will be added after all other options.
 \item Auxiliary options will only show upon entering ``Advanced Mode'' in the picker (usually by pressing ``Space'').
 \end{itemize}
 
 The boot process is as follows:
 \begin{itemize}
 \tightlist
-\item Try looking up first valid primary option through \texttt{BootNext} UEFI variable.
-\item On failure looking up first valid primary option through \texttt{BootOrder} UEFI variable.
+\item Look up the first valid primary option in the \texttt{BootNext} UEFI variable.
+\item On failure, look up the first valid primary option in the \texttt{BootOrder} UEFI variable.
 \item Mark the option as the default option to boot.
 \item Boot option through the picker or without it depending on the \texttt{ShowPicker} option.
 \item Show picker on failure otherwise.
 \end{itemize}
 
-\emph{Note 1}: This process is meant to work reliably only when \texttt{RequestBootVarRouting}
+\emph{Note 1}: This process will only work reliably when the \texttt{RequestBootVarRouting}
 option is enabled or the firmware does not control UEFI boot options (\texttt{OpenDuetPkg} or
-custom BDS). Without \texttt{LauncherOption} it also is possible that other operating systems
-overwrite OpenCore, make sure to enable it when planning to use them.
+custom BDS). When \texttt{LauncherOption} is not enabled, other operating systems can
+overwrite OpenCore settings and this property should therefore be enabled when planning
+to use other operating systems.
 
-\emph{Note 2}: UEFI variable boot options' boot arguments will be removed if present as they
-may contain arguments compromising the operating system, which is undesired once secure boot
-is enabled.
+\emph{Note 2}: UEFI variable ``boot options'' boot arguments will be removed, if present, as
+they may contain arguments that can compromise the operating system, which is undesirable when
+secure boot is enabled.
 
-\emph{Note 3}: Some operating systems, namely Windows, will create their boot option and
-mark it as top most upon first boot or after NVRAM Reset. When this happens default boot
-entry choice will update till next manual reconfiguration.
+\emph{Note 3}: Some operating systems, such as Windows, may create a boot option and mark it as
```

Comment:
Agree that it's 'such as', not namely. Ubuntu does the same!

Reply:
Btw, I hope it's helpful even if I only pick out the odd thing that I spot? As probably not going to be able to read this one and ongoing in as much detail as I volunteered myself to do on the last.